python/ray/data/tests/test_dynamic_block_split.py

```diff
@@ -501,6 +501,9 @@ def test_block_slicing(
         ),
         override_num_blocks=num_tasks,
     ).materialize()
+
+    print(f">>> [DBG] test_block_slicing {ds._plan}")
+
```

Comment:
Remove?

Reply:
Yes, will remove once i figure out tests failing only in CI but not locally